app/src/androidTest/java/com/github/se/signify/ui/screens/home/HomeScreenTest.kt

```diff
@@ -220,4 +221,21 @@ class HomeScreenTest {
       composeTestRule.onNodeWithTag("ScrollToTopButton").performClick()
     }
   }
+
+  @Test
+  fun dictionaryIsDisplayedUsingPagersScrolling() {
+    composeTestRule.setContent { HomeScreen(navigationActions) }
+
+    ('A'..'Z').forEachIndexed { _, letter ->
+      // Click on the specific letter box by scrolling to it
+      composeTestRule.onNodeWithTag("LetterBox_$letter").performScrollTo().performClick()
+
+      // Assert that the text and corresponding sign tip are displayed
+      composeTestRule.onNodeWithTag("LetterTextDict_$letter").assertIsDisplayed()
+      composeTestRule.onNodeWithTag("SignTipBox_$letter").assertIsDisplayed()
+
+      // Scroll to top button is clicked
+      composeTestRule.onNodeWithTag("ScrollToTopButton").performClick()
+    }
+  }
 }
```

Comment:
Excellent test coverage for validating swipe-based scrolling and confirming the correct letter is displayed. This ensures a robust user experience.